### PR TITLE
Fix std out log

### PR DESCRIPTION
**Pull request by MeineZ**
Set option to log to stdout to off

### CODE REVIEW

**Review by sainsay — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -66,8 +66,11 @@ set (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /MTd")
 set (CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} /MT")
 
 # === Dependencies === $
+set(WISP_LOG_TO_STDOUT OFF)
+
 add_subdirectory(deps/WispRenderer ${CMAKE_BINARY_DIR}/wisprenderer)
 
+
```
Comment (tntmeijs) [inline CMakeLists.txt]: Unnecessary whitespace

**Review by tntmeijs — CHANGES_REQUESTED** [review]

**Review by sainsay — APPROVED** [review]

**Review by sainsay — APPROVED** [review]

**Review by tntmeijs — APPROVED** [review]
